.idea/intellij.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
```

Comment:
Is this intended to be committed? If so then I wonder why is the type JAVA_MODULE?

Reply:
IDK why intellij created it, but I'm going to add it to .gitignore